backend/src/mqttClient/mqttClient.ts

```diff
@@ -23,7 +23,7 @@ import { Maybe } from './types';
 export let client: MqttClient;
 
 const mqttClient = () => {
-  client = connect('mqtts://localhost:8883');
+  client = client = connect('mqtts://localhost:8883', {rejectUnauthorized: false})
```

Comment:
There might be something else going on here, as `rejectUnauthorized: false` does not only allow signed certificates, it allows _any_ certificate to be used (according to [this thread](https://github.com/mqttjs/MQTT.js/issues/1077) at least). 

Did you try removing the certificate and generating it again? With the current main branch I was able to connect successfully without this change (but with the other commit with the SQL variable name change included).

Reply:
I just cloned it this morning, and I was not able to get it run, until I included this like, I was having a tls error on the server

Reply:
@rmelotte @Dracks can you confirm than passing the `UID/GID` to docker fix the bug and you where able to run the app without this fix?

Reply:
I confirm I can run it on my side without `rejectUnauthorized` yes (but with the fix to the SQL variable names included and the UID/GID set).

As a sidenote regarding the UID/GID, it's also possible to add it to a `.env` file in the top level directory to avoid the need to add it to every docker-compose command (without it, chances are that I would forget to set it some of the time).
Would it be worth adding a note about it in the README?

Reply:
> As a sidenote regarding the UID/GID, it's also possible to add it to a .env file in the top level directory to avoid the need to add it to every docker-compose command (without it, chances are that I would forget to set it some of the time).
Would it be worth adding a note about it in the README?

Yes I'll rework the readme. :+1: 


@slade87  could you revert this change: 

```suggestion
  client = client = connect('mqtts://localhost:8883');
```